### PR TITLE
homepage design

### PR DESCRIPTION
**Pull request by kathryngo**
responsive homepage design for zanata.org using bootstrap 4 alpha 6. it refers to remote scripts, which I will fix once I add the other pages, prior to site going live.

**Only index.html and zanata.css in root need review as the rest have been added from the bootstrap 4 library.**

### CODE REVIEW

File: zanata.css [hunk 1]
```diff
@@ -0,0 +1,329 @@
+html {
```
Comment (kathryngo) [inline zanata.css]: review this file please

File: index.html [hunk 2]
```diff
@@ -0,0 +1,225 @@
+<!DOCTYPE html>
```
Comment (kathryngo) [inline index.html]: review this file please

File: README.md [hunk 3]
```diff
@@ -1,2 +1,30 @@
-# zanata-dot-org
```
Comment (kathryngo) [inline README.md]: only review and not mark as approved automatically if you feel the need

**Comment by aeng** [comment]
@kathryngo I think this is easier to review if you got a running website rather than going through the source.

**Comment by kathryngo** [comment]
@aeng gotcha.
You can grab the branch and just open the index.html file to review this if that's easiest.

**Comment by aeng** [comment]
@kathryngo And I dont think you need a new project just for this. A new branch in https://github.com/zanata/zanata.github.io should do the trick.

**Comment by kathryngo** [comment]
👍 Aha, was going to ask when I get in if this was the best approach. I will close this branch, and delete this project and set up a new branch on github.io

**Pull request closed without merging**